browser.js

```diff
@@ -16,7 +16,7 @@ var Bootstrapper = require('./lib/bootstrapper').Bootstrapper;
  * @header boot(app)
  */
 
-exports = module.exports = function bootBrowserApp(app, options) {
+exports = module.exports = function bootBrowserApp(app, options, callback) {
```

Comment:
I want to do this incrementally. The current implementation is ready for LoopBack 3.0. We can further clean it up for LoopBack.next if we decide to drop callback style.